### ISSUE TITLE
BSC node stuck on 0 blocks won't sync with mainnet

### ISSUE THREAD

**Issue by FomoApe** [binance-chain/bsc#310]
#### System information
```
Geth
Version: 1.1.0-beta
Git Commit: 6ce2cef4250620d22324f0f74d1c1886487c0536
Git Commit Date: 20210610
Architecture: amd64
Go Version: go1.16.5
Operating System: linux
GOPATH=/root/go
GOROOT=go
```

```
Distributor ID: Ubuntu
Description:    Ubuntu 20.10
Release:        20.10
Codename:       groovy
# DISK I/O speed
read: IOPS=366k, BW=1429MiB/s (1499MB/s)(2014GiB/1442848msec)"
# Network performance
Download Mbps 922.86, Upload Mbps 897.66

4 TB SSD Nvme
128 GB RAM
```

#### Expected behaviour
I expected my node to sync to GETH

#### Actual behaviour
The node does not sync

#### Steps to reproduce the behaviour
Setup a new machine and follow official instructions from Binance which can be found [here](https://docs.binance.org/smart-chain/developer/fullnode.html). 

Commands ran

```
git clone https://github.com/binance-chain/bsc
cd bsc
make geth
wget https://github.com/binance-chain/bsc/releases/download/v1.1.0-beta/mainnet.zip
unzip mainnet.zip
#Write genesis state locally
geth --datadir node init genesis.json
#Start full node
geth --config ./config.toml --datadir ./node  --cache 18000 --rpc.allow-unprotected-txs --txlookuplimit 0
```

You can find both my config.toml file and Genesis.json files in this Gist https://gist.github.com/FomoApe/e9ad0c930e444b7b859e984b20e4cfd4

**Comment by FomoApe** [comment]
INCLUDED bsc.logs in GIST https://gist.github.com/FomoApe/e9ad0c930e444b7b859e984b20e4cfd4

**Comment by FomoApe** [comment]
Any chance you can give me tips @zhongfu, my node is pretty fast and uses SSD nvme. Could it be that my node is syncing but I'm viewing the progress of the sync wrongly?

**Comment by n8twj** [comment]
I see absolutely no errors here.   What exactly is your problem?   You node
is in the process of syning.   BTW, if you do not have a full speed,
non-shared NVMe drive, your shit will NEVER sync up.

**Comment by FomoApe** [comment]
Maybe I'm just having syncing anxiety, but it seems like my syncing is broken. Take a look at this 

```
currentBlock: 6963118,
highestBlock: 8891697,
```

The current `highestBlock` is `8892285` on bscscan and my highest block has not changed in the last hour.

**Comment by FomoApe** [comment]
Also this is a dedicated server, only used by me, so I believe the NVMe is not shared. And it's operating at near enough it's max speed of 355-375k IOPS.

**Comment by zhongfu** [comment]
how long have you been syncing it for? a typical fast sync should take at least 6-10h, and afaict your node is syncing just fine

**Comment by FomoApe** [comment]
@zhongfu initially it had synced for 4 hours and because I thought the progress was 0 blocks, I just stop the sync and drop the whole db with `geth removedb`. I did that twice, now it's been syncing for 3 hours. Let's hope it finishes soon. 

The reason why I thought it was zero was because I was running a faulty bash script. Will try to fix that script now. I don't know why I did'nt use `geth attach`? lol

Thank you @n8twj and @zhongfu will be closing this issue now
